Resources/doc/3-listener-support.rst

```diff
@@ -184,7 +196,7 @@ Listener                    Event                  Priority
 ==========================  =====================  ========
 ``MimeTypeListener``        ``kernel.request``     200
 ``FormatListener``          ``kernel.request``     34
-``VersionListener``         ``kernel.request``     33
+``VersionListener``         ``kernel.request``     31
```

Comment:
this is a problem ..
the request listener is triggered at 32 and I understand why you now moved it to 31 because you need this for the other resolvers. however if we do this at 31, then it means that mime type based versioning will not be able to set the version as a request attribute before the routing, meaning that we loose the ability to define route conditions on the version.

as such we will need to trigger the listener twice .. once before and once after the request (ie. 32).

this however also raises the question if its even feasible to allow the configuration of multiple resolvers .. or maybe it only makes sense to allow the configuration of one resolver ... ?